woocommerce-pdf-invoices-packingslips.php

```diff
@@ -273,17 +273,18 @@ public function need_woocommerce() {
 	 * PHP version requirement notice
 	 */
 	public function required_php_version() {
-		$message = sprintf(
-			'<div class="error"><p>%s</p><p>%s <a href="%s">%s</a> %s <a href="%s">%s</a>.</p></div>',
-			__( 'WooCommerce PDF Invoices & Packing Slips requires PHP 7.1 or higher (7.1 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' ),
-			__( 'You should', 'woocommerce-pdf-invoices-packing-slips' ),
-			'http://docs.wpovernight.com/general/how-to-update-your-php-version/',
-			__( 'update your PHP version', 'woocommerce-pdf-invoices-packing-slips' ),
-			__( 'or', 'woocommerce-pdf-invoices-packing-slips' ),
-			'https://github.com/wpovernight/wcpdf-dompdf-0.8.3/releases',
-			__( 'download and install this add-on', 'woocommerce-pdf-invoices-packing-slips' ),
-		);
-	
+		$error_message	= __( 'WooCommerce PDF Invoices & Packing Slips requires PHP 5.6 or higher (7.1 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' );
+		$php_message	= __( 'We strongly recommend to %supdate your PHP version%s.', 'woocommerce-pdf-invoices-packing-slips' );
+		$add_on_message	= __( 'If you cannot upgrade your PHP version, you can download %sthis addon%s to enable backwards compatibility with PHP5.6.', 'woocommerce-pdf-invoices-packing-slips' );
+
+		$message = '<div class="error">';
+		$message .= sprintf( '<p>%s</p>', $error_message );
+		$message .= sprintf( '<p>'.$php_message.'</p>', '<a href="https://docs.wpovernight.com/general/how-to-update-your-php-version/" target="_blank">', '</a>' );
+		if ( version_compare( PHP_VERSION, '5.6', '<' ) ) {
```

Comment:
5.6 is NOT supported at all. The addon is for 5.6-7.1. So should be `>` I think?

Reply:
Sorry, forgot that

Reply:
Actually, 5.6 is supported too I meant below 5.6 is not supported

Reply:
so should be `>=`

Reply:
(but maybe that doesn't make a big difference because 5.6.1 would be greater than 5.6 already)